marklogic-data-hub-central/ui/src/components/entities/custom/custom-card.test.tsx

```diff
@@ -41,12 +41,22 @@ describe("Custom Card component", () => {
             queryAllByText = renderResults.queryAllByText;
             getByRole = renderResults.getByRole;
             queryAllByRole = renderResults.queryAllByRole;
+            getByLabelText=renderResults.getAllByLabelText;
```

Comment:
can you pls fix the name to match the selector used. getAll* should be named getAll